### PR TITLE
fix show name or id en submenu energise_on_one_end

### PR DESCRIPTION
**Pull request by Walid-Sahnoun**
Signed-off-by: walid-sahnoun <walid.sahnoun@rte-france.com>

### CODE REVIEW

File: src/components/menus/line-menu.js [hunk 1]
```diff
@@ -86,7 +85,8 @@ const withLineMenu =
         const getLineDescriptor = useCallback(
             (voltageLevelId) => {
                 return displayUseName
-                    ? network.getVoltageLevel(voltageLevelId).name
+                    ? network.getVoltageLevel(voltageLevelId).name ??
```
Comment (laurgarn) [inline src/components/menus/line-menu.js:88]: You don't want to keep safe with `?.name` ?

Comment (Walid-Sahnoun) [reply src/components/menus/line-menu.js]: yes, done

File: src/components/menus/line-menu.js [hunk 2]
```diff
@@ -86,7 +85,8 @@ const withLineMenu =
         const getLineDescriptor = useCallback(
             (voltageLevelId) => {
                 return displayUseName
-                    ? network.getVoltageLevel(voltageLevelId).name
+                    ? network.getVoltageLevel(voltageLevelId)?.name ??
```
Comment (sBouzols) [inline src/components/menus/line-menu.js:88]: Use `getNameOrId`
`displayUseName ? getNameOrId(network.getVoltageLevel(voltageLevelId)) : voltageLevelId`

**Review by sBouzols — CHANGES_REQUESTED** [review]

**Review by laurgarn — APPROVED** [review]

**Review by dbraquart — APPROVED** [review]
works well

**Pull request merged**